Academic authorship chapter for Guide for Communication (#2063)
alan-turing-institute/the-turing-way: book/website/communication/aa/overview.md

(communication-aa-overview)=
# Overview of Academic Authorship

A great deal of value is placed on academic authorship because a peer-reviewed journal article is still seen as the most valued output from a research project. 

> The number of articles an academic researcher is named on, and particularly whether the person is named as first author, has significant impact on their career.  

Author: ```suggestion
The number of articles an academic researcher is named on, and particularly whether the person is named as the first author, has a significant impact on their career.  
```

But who should be an author on an article? And can we move away from the idea of first author taking the most credit?

(communication-aa-overview-who)=
## Who is an author? or are they actually contributors?

Assigning names to a publication can be a tricky business. 
However, an important decision that needs to be made at the outset of a project is whether the names on the articles produced from the project are going to be authors or contributors, or a combination of both. 

The decision may essentially be out of your hands as journals and publishers have specific policies on authorship criteria. 

> Therefore, if you want to publish in a specific journal you will have to adhere to the policy and make an agreement about this within your project.

Reviewer: Maybe here or somewhere else we can add this information regarding some publishers try to make these authorship models more justified such as PAS require public disclosure regarding the contributions of each author as in [here](https://www.pnas.org/authors/editorial-and-journal-policies) or some asks for the creation of a database that track the authors's contributions as explained [here](https://www.nature.com/articles/489475a) via [altmetrics](http://altmetrics.org/manifesto/) and some require a public correspondent for the whole article that takes all the responsibility for such. Also the use of ORCID Identifier is again accepted by some publishers that would increase the traceability that might be lost due to the name, affiliation etc. changes.

A general definition of an author is seen as the person that is the creator of the written work (creator of the ideas and the actual physical writing) and there is an ownership for the author regarding the content of what is written. 

However, most research involves a team of people to make it happen. So who decides whether these contributions are worthy of authorship or are they just to be acknowledged? 

Therefore, it is still common to give authorship to only those that have actually taken part in the writing or editing of the article as well as other substantial aspects of the research. 
This often excludes large contributions to the project from technicians, undergraduate and postgraduate students, and people involved in the infrastructure of the project such as research software engineers. 
It also raises ethical issues concerning who takes credit for the work on a project.

(communication-aa-overview-first)=
## First author dominance

> Being the first author impacts that persons reputation and visibility due to a perceived view that this person makes the greatest contribution, citation indices that in the past only counted citations for the first-named author, and citations within academic articles being shortened to the first authors name with *et al* (Ref: Frandsen & Nicolaisen 2010). 

Author: ```suggestion
Being the first author impacts that person's reputation and visibility due to a perceived view that this person makes the greatest contribution, citation indices that in the past only counted citations for the first-named author, and citations within academic articles being shortened to the first author's name with *et al* (Ref: Frandsen & Nicolaisen 2010). 
```

The number of citations that papers have and what journals they are published in are often important criteria for obtaining new jobs, getting promoted and winning funding applications.

> The practice of first author taking prominence over others also has social implications. 

Author: ```suggestion
The practice of the first author taking prominence over others also has social implications. 
```

> For example, it might be a difficult decision for a researcher to change their name, due to marriage or divorce, as this action could have an negative impact on their research career. 

Author: ```suggestion
For example, it might be a difficult decision for a researcher to change their name, due to marriage or divorce, as this action could have a negative impact on their research career. 
```
